posthog/clickhouse/migrations/0101_add_exchange_rates.py

```diff
@@ -1,24 +1,20 @@
-from posthog.clickhouse.client.migration_tools import run_sql_with_exceptions
-from posthog.clickhouse.cluster import NodeRole
-from posthog.models.exchange_rate.sql import (
-    DROP_EXCHANGE_RATE_TABLE_SQL,
-    DROP_EXCHANGE_RATE_DICTIONARY_SQL,
-    EXCHANGE_RATE_TABLE_SQL,
-    EXCHANGE_RATE_DATA_BACKFILL_SQL,
-    EXCHANGE_RATE_DICTIONARY_SQL,
-)
-
-operations = [
-    # Drop tables to allow this to rerun
-    run_sql_with_exceptions(DROP_EXCHANGE_RATE_TABLE_SQL()),
-    run_sql_with_exceptions(DROP_EXCHANGE_RATE_TABLE_SQL(on_cluster=False), node_role=NodeRole.COORDINATOR),
-    run_sql_with_exceptions(DROP_EXCHANGE_RATE_DICTIONARY_SQL()),
-    run_sql_with_exceptions(DROP_EXCHANGE_RATE_DICTIONARY_SQL(on_cluster=False), node_role=NodeRole.COORDINATOR),
+# These have all been commented out to guarantee this runs
+# properly on tests
+# If we create these and shortly after run 0102 then we'll see
+# some replica errors because we've barely created the tables
+# and then immediately try to delete them which fails
+operations: list = [
+    # Drop tables/dictionaries to allow this to rerun
+    # Dict first because it depends on the table
+    # run_sql_with_exceptions(DROP_EXCHANGE_RATE_DICTIONARY_SQL()),
+    # run_sql_with_exceptions(DROP_EXCHANGE_RATE_DICTIONARY_SQL(on_cluster=False), node_role=NodeRole.COORDINATOR),
+    # run_sql_with_exceptions(DROP_EXCHANGE_RATE_TABLE_SQL()),
+    # run_sql_with_exceptions(DROP_EXCHANGE_RATE_TABLE_SQL(on_cluster=False), node_role=NodeRole.COORDINATOR),
     # Recreate them all
-    run_sql_with_exceptions(EXCHANGE_RATE_TABLE_SQL()),
-    run_sql_with_exceptions(EXCHANGE_RATE_TABLE_SQL(on_cluster=False), node_role=NodeRole.COORDINATOR),
-    run_sql_with_exceptions(EXCHANGE_RATE_DATA_BACKFILL_SQL()),
-    run_sql_with_exceptions(EXCHANGE_RATE_DATA_BACKFILL_SQL(), node_role=NodeRole.COORDINATOR),
-    run_sql_with_exceptions(EXCHANGE_RATE_DICTIONARY_SQL()),
-    run_sql_with_exceptions(EXCHANGE_RATE_DICTIONARY_SQL(on_cluster=False), node_role=NodeRole.COORDINATOR),
+    # run_sql_with_exceptions(EXCHANGE_RATE_TABLE_SQL()),
+    # run_sql_with_exceptions(EXCHANGE_RATE_TABLE_SQL(on_cluster=False), node_role=NodeRole.COORDINATOR),
+    # run_sql_with_exceptions(EXCHANGE_RATE_DATA_BACKFILL_SQL()),
+    # run_sql_with_exceptions(EXCHANGE_RATE_DATA_BACKFILL_SQL(), node_role=NodeRole.COORDINATOR),
+    # run_sql_with_exceptions(EXCHANGE_RATE_DICTIONARY_SQL()),
+    # run_sql_with_exceptions(EXCHANGE_RATE_DICTIONARY_SQL(on_cluster=False), node_role=NodeRole.COORDINATOR),
```

Comment:
Is it ok to just comment them all? I guess it doesn't really hurt, but feels weird.